src/Pages/Contact/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import { useState, useEffect } from "react";
```

Comment:
You do not use this value in your request data

So set it to `undefined`

<img width="153" alt="Screenshot 2023-01-09 at 21 43 26" src="https://user-images.githubusercontent.com/34886027/211394252-7d97649c-c0b5-4994-8baf-c70f0228f3bb.png">

Reply:
`orderFast: undefined`